ext/configuration.h

```diff
@@ -142,6 +142,7 @@ enum ddtrace_dbm_propagation_mode {
     CONFIG(BOOL, DD_TRACE_CLIENT_IP_ENABLED, "false")                                                          \
     CONFIG(STRING, DD_TRACE_CLIENT_IP_HEADER, "")                                                              \
     CONFIG(BOOL, DD_TRACE_FORKED_PROCESS, "true")                                                              \
+    CONFIG(INT, DD_TRACE_HOOK_LIMIT, "100")                                                                   \
```

Comment:
Nitpick: there is a space missing in this line, so the `\` in the end don't line up currently
```suggestion
    CONFIG(INT, DD_TRACE_HOOK_LIMIT, "100")                                                                    \
```